docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform-2022.md

```diff
@@ -0,0 +1,225 @@
+```bash
+yarn add pg
+```
+</code-block>
+
+</code-group>
+
+### Committing the project to a remote repository
+
+Save the Strapi application locally then, in a terminal, run :
+
+1. `git add .`
+2. `git commit -m "commit message"`
+3. `git push`
+
+## Creating a DigitalOcean App
+
+On the DigitalOcean website click on the **Create** button and select *Apps*.  Next select GitHub and authorize access to the correct repository. Next:
+
+1. select the branch,
+2. select the source directory (optional),
+3. chose whether or not to "Autodeploy" when an update is pushed to the GitHub repository,
+4. click the **Next** button.
+
+### Connecting a web app to a database
+
+On the next screen:
+
+1. click **add resource**,
+2. select database, and click **Add**,
+3. On the next screen select *dev database* and name the database. The default database name is "db" and that will be used in the subsequent sections.
+4. Click the **Create and attach** button.
+5. On the next screen click the **Next** button to set the environment variables.
+
+### Environmental variables
+
+In the DigitalOcean App Platform there are global and component level environment variables. Strapi applications in production need global variables to set the database connection, and can use component level variables for secrets storage. Add the following variables to the global environment table:
+
+| Variable name     | Value          |
+|-------------------|----------------|
+| URL               | ${APP_URL}     |
+| DATABASE_HOST     | ${db.HOSTNAME} |
+| DATABASE_PORT     | ${db.PORT}     |
+| DATABASE_NAME     | ${db.DATABASE} |
+| DATABASE_USERNAME | ${db.USERNAME} |
+| DATABASE_PASSWORD | ${db.PASSWORD} |
+| NODE_ENV          | production     |
+
+Click **Save**.
+At the top of the *Settings* menu select your Strapi application component. Scroll down to the Environment variables and click **edit** on the far right of the menu. The secrets for a Strapi application are set in the component settings environment variables. In your local application these secrets are stored in the `.env` file by default. Security is enhanced if different keys are generated for different environments. Add the following secrets and keys to the component environment variables table:
+
+| Variable name    | value                               |
+|------------------|-------------------------------------|
+| APP_KEYS         | "unique user-generated secret here" |
+| API_TOKEN_SALT   | "unique user-generated secret here" |
+| ADMIN_JWT_SECRET | "unique user-generated secret here" |
+| JWT_SECRET       | "unique user-generated secret here" |
+
+Click **Save**.
+
+::: caution
+The environmental variables for the development and managed database are handled differently by DigitalOcean. When connected to a managed database DigitalOcean passes the environmental variables automatically as a url. Check the .yaml file to confirm the db settings are not duplicated if using a managed database.
```

Comment:
Again, careful with the spelling of some words:
- "url" should be written "URL"
- "db settings" shouldn't it be "database settings"?

Also, shouldn't ".yaml" be in code formatting since that's what you did most of the time in that documentation? :)